semcore/data-table/src/Head.tsx

```diff
@@ -216,7 +216,7 @@ class Head extends Component<AsProps> {
             {column.props.children}
             {column.sortable ? (
               <SSortWrapper ref={this.makeSortRefHandler(column.active)}>
-                <SSortIcon active={column.active} />
+                <SSortIcon />
```

Comment:
I've removed it, because we can't pass this prop because of types, also I couldn't find a reason, for what this prop were using